forte/processors/stave/stave_processor.py

```diff
@@ -0,0 +1,373 @@
+            url(r'.*/static/(.*)', StaticFileHandler, {
+                "path": build_path + "/static/"
+            }),
+            url(r'.*/([^/]*\.png)', StaticFileHandler, {"path": build_path}),
+            url(r'.*/([^/]*\.ico)', StaticFileHandler, {"path": build_path}),
+            url(r"/.*", self.ReactHandler, {"build_path": build_path})
+        ])
+
+        server = HTTPServer(app)
+        server.listen(self.configs.port)
+
+        # Release lock in main thread
+        self._barrier.wait()
+
+        IOLoop.current().start()
+
+    def _create_project(self, input_pack: DataPack):
+        with requests.Session() as session:
+
+            # Log in as admin user
+            response = session.post(f"{self._url}/api/login",
+                json={
+                    "name": self.configs.user_name,
+                    "password": self.configs.user_password
+                })
+            logger.info("%d %s", response.status_code, response.text)
+            if response.status_code != 200:
+                return
+
+            # Configure and create project
+            if self._project_id < 0:
+                response = session.post(f"{self._url}/api/projects/new",
+                    json={
+                        "type": self.configs.projectType,
+                        "name": self.configs.projectName,
+                        "ontology": json.dumps(
+                            self.resources.get("onto_specs_dict")
+                        ),
+                        "multiOntology": str(self.configs.multiOntology),
+                        "config": str(self.configs.projectConfigs)
+                    })
+                logger.info("%d %s", response.status_code, response.text)
+                if response.status_code != 200:
+                    return
+                self._project_id = response.json()["id"]
+                webbrowser.open(self._url)
+
+            # Configure and create document
+            response = session.post(f"{self._url}/api/documents/new",
+                json={
+                    "name": input_pack.pack_name,
+                    "textPack": input_pack.serialize(),
+                    "project_id": self._project_id,
+                })
+            logger.info("%d %s", response.status_code, response.text)
+            if response.status_code != 200:
+                return
+
+    @classmethod
+    def default_project_configs(cls, ontology: Dict[str, Any]):
+        """
+        Create default project configuration based on ontology.
+        This is translated from JavaScript function `createDefaultConfig` in
+        https://github.com/asyml/stave/blob/d82383de3d74bf09c0d30f33d8a902595
+        f5aff80/src/app/pages/Projects.tsx#L140
+        Commit link:
+        https://github.com/asyml/stave/commit/63ae90583016bbc7d88c091a3bd8a2b
+        56675e49e#diff-46d2dc8ce1bf31f277e7a217416c0be739924160096aac4ca1fee4
+        23bee4c014R105
```

Comment:
These links lead to 404 on my end. I think it is because of the line break. You can simply write the URL in one line, and ignore line length check in pylint (for the scope of this function.)

Also, the permanent link to the function is fine, we don't need the commit link here.